r/R/expression.R

```diff
@@ -100,7 +100,8 @@
   # use `%/%` above.
   "%%" = "divide_checked",
   "^" = "power_checked",
-  "%in%" = "is_in_meta_binary"
+  "%in%" = "is_in_meta_binary",
+  "strrep" = "binary_repeat"
```

Comment:
`strrep()` is a function in base R. There is also a function [`str_dup()`](https://stringr.tidyverse.org/reference/str_dup.html) in the popular R package **stringr** that does exactly the same thing. In the R bindings we often like to add these **stringr** variants of the functions too:
```suggestion
  "strrep" = "binary_repeat",
  "str_dup" = "binary_repeat"
```

Reply:
Good to know. Thanks!

Reply:
There is a minor difference in behavior when given a negative repeat count:
* `binary_repeat` and `strrep` return an error
* `str_dup` returns `NA`